src/SprykerSdk/Zed/Integrator/Communication/Plugin/Composer/InstallerComposerPlugin.php

```diff
@@ -133,6 +133,11 @@ public function collect(PackageEvent $event): void
      */
     public function runInstaller(Event $event): void
     {
+        //Disable installer on shop pre-install phase
+        if (!class_exists(ModuleTransfer::class)) {
```

Comment:
i suggest to add `\Generated\Shared\Transfer\ClassInformationTransfer::CLASS_TOKEN_TREE` const is defined check here.